meta_client/src/types.rs

```diff
@@ -298,20 +305,29 @@ impl From<CreateTableRequest> for meta_service::CreateTableRequest {
             header: None,
             schema_name: req.schema_name,
             name: req.name,
-            create_sql: req.create_sql,
+            encoded_schema: req.encoded_schema,
+            engine: req.engine,
+            create_if_not_exist: req.create_if_not_exist,
+            options: req.options,
         }
     }
 }
 
-impl From<meta_service::CreateTableResponse> for CreateTableResponse {
-    fn from(pb_resp: meta_service::CreateTableResponse) -> Self {
-        Self {
-            schema_name: pb_resp.schema_name,
-            name: pb_resp.name,
-            shard_id: pb_resp.shard_id,
-            schema_id: pb_resp.schema_id,
-            id: pb_resp.id,
-        }
+impl TryFrom<meta_service::CreateTableResponse> for CreateTableResponse {
+    type Error = Error;
+
+    fn try_from(pb_resp: meta_service::CreateTableResponse) -> Result<Self> {
+        let pb_table_info = pb_resp.created_table.context(MissingTableInfo {
+            msg: "created table is not found in the create table response",
```

Comment:
table requested to create is not found in the response ?

Reply:
Yes. Actually, this won't happen in the real implementation, but the protobuf message generated by prost make any sub message is optional which leading to such check here.